tests/unit/specs/game-state.spec.js

```diff
@@ -101,7 +101,7 @@ describe('Converting GameState, and GameStateRow, and Publishing Socket for Reso
     expect(gameState).toEqual(resolveNine.gameState);
   });
 
-  it('Creates and Publishes socket event', async () => {
+  it.only('Creates and Publishes socket event', async () => {
```

Comment:
Oops.
```suggestion
  it('Creates and Publishes socket event', async () => {
```

Reply:
Fun fact: vitest errors on this in CI unless you explicitly allow it with a command line option. Pretty sweet feature